src/commands/ai/mixtral.ts

```diff
@@ -0,0 +1,105 @@
+import type { Command } from '@commands';
+import { SlashCommandBuilder } from 'discord.js';
+
+import { replicate } from './replicate';
+
+/*
+const input = {
+    prompt: "Write a bedtime story about neural networks I can read to my toddler",
+    max_new_tokens: 1024,
+    temperature: 0.6,
+    top_p: 0.9,
+    top_k: 50,
+    presence_penalty: 0,
+    frequency_penalty: 0
+    prompt_template: "<s>[INST] {prompt} [/INST] ",
+};
+*/
+
+export const MIXTRAL: Command = {
+    data: new SlashCommandBuilder()
+        .setName('mixtral')
+        .setDescription('Génère du texte avec Mixtral 8x7b!')
+        .addStringOption((option) => option.setName('prompt').setDescription('Le prompt').setRequired(true))
+        .addNumberOption((option) =>
+            option
+                .setName('max_new_tokens')
+                .setDescription('Le nombre maximum de jetons que le modèle doit générer en sortie (1024 par défaut)')
```

Comment:
the other command (ping) is in english, do we do 100% english, french, localized ?

Reply:
The server we target for is in french.
So either we do both with localization feature, either all in french. The french language is **mandatory**.

Reply:
k will do another PR to verify all of this